.github/workflows/gemini-cli.yml

```
@@ -0,0 +1,304 @@
+name: '💬 Gemini CLI'
+
+on:
+  pull_request_review_comment:
+    types:
+      - 'created'
+  pull_request_review:
+    types:
+      - 'submitted'
+  issue_comment:
+    types:
+      - 'created'
```

The `concurrency` setting can be simplified. Instead of using a complex condition, you can directly use the `github.head_ref` for pull requests and `github.ref` for other events. This is a more standard and readable way to set concurrency.
```suggestion
concurrency:
  group: '${{ github.workflow }}-${{ github.head_ref || github.ref }}'
  cancel-in-progress: true
```